OWNERS

```diff
@@ -1,19 +1,15 @@
 reviewers:
   - aburdenthehand
-  - cwilkers
   - dhiller
   - jean-edouard
-  - jobbler
 
 approvers:
   - aburdenthehand
-  - AlonaKaplan
-  - cwilkers
   - davidvossel
   - fabiand
+  - jean-edouard
   - rmohr
   - vladikr
```

Comment:
nit: this is redandunt

Reply:
Respectfully disagree - it returns the list to an alphabetised state.